app/controllers/students_controller.rb

```diff
@@ -0,0 +1,16 @@
+class StudentsController < ApplicationController
+  before_action :set_student, only: :show
```

Comment:
I'll fix this, but they won't know the before_action stuff yet so we shouldn't use it